app/data_models/list_store.py

```diff
@@ -137,6 +137,9 @@ def _build_map(list_models: List[Mapping]):
     def get(self, item: str):
         return self.__getitem__(item)
 
+    def list_item_index(self, for_list: str, list_item_id: str) -> int:
+        return self[for_list].index(list_item_id)
```

Comment:
Given we can get the index of a list item by doing something like:
```python
list_store[for_list].index(list_item_id)
```

And our primary usage of this is to get the index plus one, we should consider changing this to a method to do that, something like `list_item_position`?

Reply:
Done